taxi/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    country = models.CharField(max_length=50)
+
+    def __str__(self):
+        return f"{self.name}, {self.country}"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=50, unique=True)
+    username = models.CharField(max_length=50, unique=True)
+    email = models.EmailField(max_length=50)
+    password = models.CharField(max_length=50)
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+
+    def __str__(self):
+        return f"{self.username} {self.first_name} {self.last_name}"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=50)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name='cars'
+    )
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name='cars')
+
+    def __str__(self):
+        return f"CAR:{self.model}, manufacturer:{self.manufacturer}, drivers:{self.drivers}"
```

Comment:
You are overusing f-string. It's not a big issue, but it would be better to simply return the attribute without wrapping it in an f-string. For example: `return self.model`. This is more about code style and readability (Checklist item 3).